docs/release_process.md

```diff
@@ -0,0 +1,25 @@
+# Release process
+
+## Frequency
+
+Open Service Mesh will target a release every quarter. These releases will be targeted in the first half of February, May, August, and November. These target release months are subject to change if a release is pushed. In other words, if a release misses February and is pushed out in March then the next release will target June instead of May.
+
+## Process for requesting a change
+
+If you would like to see a new feature added to Open Service Mesh, please [open an issue](https://github.com/openservicemesh/osm/issues). This issue will be discussed and, if approved, prioritized in the backlog.
+
+## Creating a new release
+
+When there is a new release being planned and developed, a [milestone](https://github.com/openservicemesh/osm/milestones) will be created. This new milestone will have issues added to it that will be prioritized from the [vFuture milestone](https://github.com/openservicemesh/osm/milestone/32). Once the release milestone is complete, a new release of Open Service Mesh will be created and this milestone will be closed. At that moment, the next release milestone will be created and items from vFuture will be added.
```

Comment:
I'd be interested in discussing this prioritization process. Do we have any documentation around that?

Reply:
The prioritization process is a combination of roadmap influence as well as roundtable discussion with the maintainers and the community.

Reply:
if any of those conversations/roadmap are public, it may be worth including links, dates, etc in this doc